### PR TITLE
SAK-29791 Remove covers from /assignment

### PR DESCRIPTION
**Pull request by buckett**
The assignment API/Cover had different method signatures and so there are a few places where exceptions would never have been thrown and now the API doesn't declare them so we don't need the catch statements any more.

### CODE REVIEW

File: assignment/assignment-bundles/resources/assignment_eu.properties [hunk 1]
```diff
@@ -697,12 +697,12 @@ audio.invalid_number_of_attempts=Mesedez, adierazi ikasleek zenbat alditan haien
 audio.number_of_attempts.noselect=Aukeratu...
 audio.number_of_attempts.unlimited=Mugarik gabe
 gen.theassall7 = Zeregin honetan audio grabaketa soilik erabili daiteke. 
-Audio grabaketa txertatzeko sakatu �Grabatu� botoia.  
-gen.theassall8 = Zeregin honetan audio grabaketa eta testu editorea erabili daitezke. Idatzi zure iruzkinak testu eremuan, eta �Grabatu� botoia sakatu audio grabaketa bat txertatzeko. 
+Audio grabaketa txertatzeko sakatu �Grabatu� botoia.
```
Comment (ottenhoff) [inline assignment/assignment-bundles/resources/assignment_eu.properties]: This change looks suspicious

File: assignment/assignment-bundles/resources/assignment_eu.properties [hunk 2]
```diff
@@ -751,7 +751,7 @@ failed= Eragiketa ezin izan da burutu.
 notpermis4= Baliabideak gehitzeko behar diren baimenak ez dituzu. 
 processmessage.file= Fitxategia(k) igotzen. 
 singleatt = Fitxategi bat soilik
-size.exceeded= �{0}� fitxategiaren neurria baimentzen dena baina handiagoa da.
+size.exceeded= �{0}� fitxategiaren neurria baimentzen dena baina handiagoa da.
```
Comment (buckett) [inline assignment/assignment-bundles/resources/assignment_eu.properties]: And this one looks wrong too.

**Comment by buckett** [comment]
Closing for now as I'm not going to manage to fix this soon.

File: assignment/assignment-bundles/resources/assignment_eu.properties (hunk 1 above)
Comment (buckett) [inline assignment/assignment-bundles/resources/assignment_eu.properties]: Yep, good spot.

**Comment by buckett** [comment]
`youarenot_viewSubmission_1` and `cannotfin_submission_1` were the 2 properties keys I was trying to remove.

**Pull request closed without merging**